src/utils.rs

```diff
@@ -25,7 +25,7 @@ fn storage_key_hash_vec(module: &str, storage_key_name: &str, param: Option<Vec<
     let mut key = [module, storage_key_name].join(" ").as_bytes().to_vec();
     match param {
         Some(par) => {
-            key.append(&mut par.clone());
+            key.extend(&par.clone());
```

Comment:
Tiny detail; I believe `par` is cloned twice as extend does implicitly apply clone.